pkgs/development/libraries/libgdiplus/default.nix

```
@@ -1,18 +1,23 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, glib, cairo, Carbon, fontconfig
+{ lib, stdenv, fetchzip, pkg-config, glib, cairo, Carbon, fontconfig
 , libtiff, giflib, libjpeg, libpng
 , libXrender, libexif, autoreconfHook, fetchpatch }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (self: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
that's the standard name we use

---

Thanks, done.